.github/workflows/docs.yaml

```diff
@@ -1,4 +1,5 @@
-name: deploy-docs
+---
+name: "Deploy docs"
 
 # Only run this when the master branch changes
```

Comment:
any way to allow doc preview for prs, or maybe do the doc build without the push, to check build is not broken? I am thinking about this UI element on github

![image](https://user-images.githubusercontent.com/442720/93471321-7e6c7580-f8f3-11ea-946e-d0ac83873bc2.png)

Reply:
Yes, I can add build-docs without pushing to gh-pages I hope that is a better approach than pushing all changes from the PR's. It's ok?

Reply:
sounds good

Reply:
Okay, so we have now built docs with tox. For every PR and master push is uploaded docs/_build/html as docs_build zip artifact.